public/style/sass/_reset.scss

```diff
@@ -1,15 +1,16 @@
 /* SIZING */
 
-body, div, header, section, h1, button, label {
+body, div, header, section, h1, select, label, button, input, textarea {
   margin: 0;
   padding: 0;
   border: 0;
+  border-radius: 2px;
```

Comment:
Could you move border-radius into your report scss file? This section is just for resetting to default values, rather than adding styling